waffle-compiler/test/flattener/testSource/cycle1.sol

```diff
@@ -0,0 +1,5 @@
+import "./cycle2.sol";
```

Comment:
Every solidity file needs to have a `pragma` statement. I'd add them to these test files to make the test more realistic.